bolt-design-system/bolt
# Trigger | FE | Add custom cursor and pointer as fallback for zoom-in and zoom-out in IE11

## Jira

http://vjira2:8080/browse/BDS-1679

## Summary

Adds IE11 specific fallback for `zoom-in` and `zoom-out` options for `bolt-trigger`'s cursor prop.

## Details

IE11 targeted CSS are written for when the cursor is defined as `zoom-in` or `zoom-out`.

## How to test

Run the branch locally and check the trigger cursor demo page in IE11. Make sure the cursor for `zoom-in` and `zoom-out` options are showing the `pointer` cursor (finger).
